### ISSUE TITLE
[WEB-CLIENT] Email sign-up

### ISSUE THREAD

**Issue by isma-osy** [reach4help/reach4help#788]
**Is your feature request related to a problem? Please describe.**
Providing additional options for the sign-up

**Describe the solution you'd like**
On the sign-up screen, provide the option for the user to sign-up by using his Name and email
to sign-up. This is additional to the Facebook and Google login options. 
Flow:
-The user inputs his name and email on the sign-up screen
-An email is sent to to the email adress he provided, it contains a verification link
-The user clicks on the verification link to verify his email, if this succeeds he is signed-up
Screens:
![Screenshot 2020-06-22 at 17 30 22](https://user-images.githubusercontent.com/1456473/85425298-ea4b7600-b570-11ea-9fcf-e95e82515e9d.png)

--------
### [ZenHub Information](https://app.zenhub.com/workspaces/reach4help-5e8dcbfb14ac087f410cbabb/issues/reach4help/reach4help/788)

*This information is updated automatically. To modify it, please use ZenHub.*

--------
### [ZenHub Information](https://app.zenhub.com/workspaces/reach4help-5e8dcbfb14ac087f410cbabb/issues/reach4help/reach4help/788)

*This information is updated automatically. To modify it, please use ZenHub.*

**Belonging to Epics:**

* **[EPIC]** [#804 - [WEB-CLIENT] Sign-up simplification and usability upgrade](https://github.com/reach4help/reach4help/issues/804)

**Comment by mcverter** [comment]
@isma-osy @schowdhury8 

I am rejecting this issue because it does not build off of the current screens. 
You are giving me too many screens to fix.  
I am only willing to change one screen per issue.

If you want screens to be reordered open up a separate issue for their reordering once everything is done

**Comment by mcverter** [comment]
![AUTH_STEP_1](https://user-images.githubusercontent.com/1453956/85439786-5fac4c00-b55b-11ea-8129-9a845b951b53.png)
![AUTH_STEP_2](https://user-images.githubusercontent.com/1453956/85439788-5fac4c00-b55b-11ea-8b9f-447eabcb80d4.png)
![AUTH_STEP_3](https://user-images.githubusercontent.com/1453956/85439791-6044e280-b55b-11ea-8171-63bd437d7b7a.png)
![AUTH_STEP_4_POPUP](https://user-images.githubusercontent.com/1453956/85439792-6044e280-b55b-11ea-8414-b02ad1a7f353.png)
![AUTH_STEP_4](https://user-images.githubusercontent.com/1453956/85439793-6044e280-b55b-11ea-8ba9-2a0ca6f2ad6c.png)
